* fix location of libXfont2.so and libfontenc.so
* don't derive location of libpixman-1.so from path to libX11.so

author: Kenneth Hoste (HPC-UGent)
--- turbovnc-2.2.3/unix/Xvnc/CMakeLists.txt.orig	2019-10-11 13:27:06.554563048 +0200
+++ turbovnc-2.2.3/unix/Xvnc/CMakeLists.txt	2019-10-11 13:40:52.455275026 +0200
@@ -92,9 +92,8 @@
 if(TVNC_SYSTEMX11)
 	include_directories(${X11_X11_INCLUDE_PATH} ${X11_Xau_INCLUDE_PATH}
 		${X11_Xdmcp_INCLUDE_PATH} ${X11_Xkbfile_INCLUDE_PATH})
-	string(REGEX REPLACE "X11" "Xfont2" X11_Xfont2_LIB ${X11_X11_LIB})
-	string(REGEX REPLACE "X11" "fontenc" X11_Fontenc_LIB ${X11_X11_LIB})
-	string(REGEX REPLACE "X11" "pixman-1" X11_Pixman_LIB ${X11_X11_LIB})
+	string(REGEX REPLACE "libX11" "libXfont2" X11_Xfont2_LIB ${X11_X11_LIB})
+	string(REGEX REPLACE "libX11" "libfontenc" X11_Fontenc_LIB ${X11_X11_LIB})
 else()
 	include_directories(${CMAKE_CURRENT_BINARY_DIR}/X_include)
 	set(X11_Xau_LIB Xau)
